test/e2e/catalog_e2e_test.go

```diff
@@ -78,9 +80,18 @@ var _ = Describe("Catalog represents a store of bundles which OLM can use to ins
 			},
 		}
 
-		crd := newCRD(genName("ins"))
+		crd := newCRD(genName("ins-"))
 		csv := newCSV(packageStable, ns.GetName(), "", semver.MustParse("0.1.0"), []apiextensions.CustomResourceDefinition{crd}, nil, nil)
 
+		defer func() {
+			Eventually(func() error {
+				return ctx.Ctx().KubeClient().ApiextensionsInterface().ApiextensionsV1().CustomResourceDefinitions().Delete(context.TODO(), crd.GetName(), metav1.DeleteOptions{})
```

Comment:
nit: context.TODO -> context.Background.

Reply:
nit: `return client.IgnoreNotFound(...)` so we can avoid having to use any complex gomega matching expressions?

Reply:
(I realize I'm making the assumption that this is client utility is still applicable for non-controller-runtime clients)

Reply:
Fixed. Looking at the implementation of `IgnoreNotFound` it's exactly what you'd expect lol ^^ so should work with any err